src/styles/charts/loading.scss

```diff
@@ -0,0 +1,15 @@
+
+$animation-duration: 1.5s;
+$animation-from: -66.66%;
+$animation-to: 0;
+
+.line-load-state--css-wave .chart-filter,
```

Comment:
This 'css-wave' is set as a variant, is it because we want to add different styles, so having different modes?

Reply:
That would be a reason to do this, but for now we can probably just apply this style to `.line-load-state`

Reply:
yeah, probably it would be easier for now